src/libstd/lib.rs

```diff
@@ -469,6 +469,7 @@ pub mod error;
 pub mod ffi;
 pub mod fs;
 pub mod io;
+#[cfg(not(target_os = "l4re"))]
```

Comment:
It's ok to not have networking implemented, Emscripten for example doesn't implement threads. To do this, however, the shims in the `sys` module should all return errors on l4re instead of being compiled away.